wicket-jquery-ui/src/main/java/com/googlecode/wicket/jquery/ui/interaction/sortable/SortableBehavior.java

```diff
@@ -87,15 +86,26 @@ public SortableBehavior(String selector, Options options, ISortableListener<T> l
 	protected abstract List<T> getItemList();
 
 	/**
-	 * Gets the list of the connected sortable
+	 * Gets the list of the connected sortable.
 	 *
 	 * @return null by default
 	 */
+	@Deprecated
```

Comment:
javadoc

Reply:
I'm currently reviewing, I will manage the javadoc...